src/pages/Environments.js

```diff
@@ -638,7 +510,7 @@ export const Environments = ({ nav = undefined }) => {
               }
             },
             {
-              size: { min: '7em', grow: 0 },
+              size: { min: '8em', grow: 0 },
```

Comment:
Needed to increase to avoid wrapping between status and icon (and the same column in the other table was already at 8em. I think this may be changing again though to 10 in another PR.